src/travel-details-screens/components/TripSection.tsx

```diff
@@ -108,6 +119,27 @@ export const TripSection: React.FC<TripSectionProps> = ({
     leg.toPlace.quay?.id,
   );
 
+  const publicCode = getPublicCodeFromLeg(leg);
+
+  const now = useNow(2500);
+  const {flex_booking_number_of_days_available} = useRemoteConfig();
+  const bookingIsAvailable = getLegBookingIsAvailable(
+    leg,
+    now,
+    flex_booking_number_of_days_available,
+  );
+  const requiresBookingUrgently = getLegRequiresBookingUrgently(leg, now);
+
+  const atbAuthorityId = 'ATB:Authority:2';
```

Comment:
I could consider to have this value somewhere in a way if this change. For instance: Another company has support for Flexible booking. So, moving it to a remote config or something gives that flexibility, is just a suggestion we can ask other team members what do they think.

Reply:
Yeah it could be a good idea. @gorandalum any thoughts about this?

Reply:
Update: following discussion with @gorandalum, leave it as is for now.

Reply:
Consider creating a task to fix this in the future when needed.